page-objects/pages/TASKListPage.js

```diff
@@ -75,6 +77,15 @@ async function sortTasksLastEdited() {
 	await waitHelpers.waitUntilAjaxIsFinished();
 }
 
+async function sortTasksKurse() {
+	await elementHelpers.click(filterSelect);
+	await elementHelpers.click(kurseSelect);
+	await elementHelpers.clickAndWait(kurseCheckbox);
+	await elementHelpers.clickAndWait(submitBtn);
+	await waitHelpers.waitUntilPageLoads();
+	await waitHelpers.waitUntilAjaxIsFinished();
```

Comment:
Do you know if ajax is used in that part of the application? If not, we can remove the method.